AutoRest/Generators/Ruby/Azure.Ruby/AzureRubyCodeGenerator.cs

```diff
@@ -106,7 +187,22 @@ public static void CorrectFilterParameters(ServiceClient serviceClient)
             }
 
             // Models
-            foreach (var model in serviceClient.ModelTypes)
+
+            // Paged Models
```

Comment:
PageTemplateModel being an special type of ModelTemplateModel. can we hide details somewhere to make model writtting more general here?